modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase

```diff
@@ -38,24 +38,6 @@ definition {
 
 	}
 
-	@description = "Verify that any user with administrator role can reassign allowed tasks"
```

Comment:
Removed as it is already covered in another test https://liferay.atlassian.net/browse/LRQA-74797